consensus/consensus_fbft.go

```diff
@@ -0,0 +1,30 @@
+package consensus
+
+import "sync"
+
+type LockedFBFTPhase struct {
+	mu     sync.Mutex
+	phrase FBFTPhase
```

Comment:
- It might be a good idea to use RWMutex which allows to other threads also be able to read the value
- do you mean phrase or phase? is it typo?

Reply:
FBFTPhase is a string, immutable struct, so only copy is possible to it. Copy takes less than 0.5ns, lock takes about  15ns. No reason to grant a shared access to smth that lasts less than lock getting time. It's a good idea to reduce lock getting time, and `sync.Mutex` is the most simplest and fastest.

Reply:
"phrase FBFTPhase"   phrase should be phase, right?

Reply:
you mean `FBFTphase` or what?